gprotorch/gp.py

```diff
@@ -0,0 +1,131 @@
+
```

Comment:
Should we have a docstring at the top describing what the module is doing and listing you as the author @InfProbSciX !

Reply:
Yep sorry, need to add docstrings